profiles/cinnamon.py

```diff
@@ -10,8 +10,7 @@ def _prep_function(*args, **kwargs):
 	for more input before any other installer steps start.
 	"""
 
-	# Cinnamon optionally supports xorg, we'll install it since it also
-	# includes graphic driver setups (this might change in the future)
+	# Cinnamon requires a functioning Xorg installation.
```

Comment:
For reference: https://github.com/linuxmint/cinnamon/issues/8512#issuecomment-482975669